### PR TITLE
advanced search added

### PR DESCRIPTION
**Pull request by namjohn920**
advanced search added.
Related to #1632

### CODE REVIEW

File: site/pages/search/index.hbs [hunk 1]
```diff
@@ -404,6 +405,15 @@
 &lt;/div&gt;
 </code></pre>
 
+<h3>Advanced Search Link</h3>
+<p>Advanced search link below the pagination tab will allow users to enhance the search result.</p>
+
+<pre><code>
+&lt;div&gt;
```
Comment (duboisp) [inline site/pages/search/index.hbs]: No need for a div's here

File: site/pages/search/results-en.hbs [hunk 2]
```diff
@@ -200,3 +200,8 @@
 		</ul>
 	</div>
 </section>
+
+{{!-- Advanced search link --}}
```
Comment (duboisp) [inline site/pages/search/results-en.hbs]: Remove this comment in the working example.

File: site/pages/search/results-en.hbs [hunk 3]
```diff
@@ -200,3 +200,8 @@
...
+<div>
```
Comment (duboisp) [inline site/pages/search/results-en.hbs]: Remove this empty div.

File: site/pages/search/index.hbs [hunk 4]
```diff
@@ -404,6 +405,15 @@
...
+	&lt;p&gt;&lt;a href="https://www.canada.ca/en/sr/srb/sra.html"&gt;Advanced search&lt;/a&gt;&lt;/p&gt;
```
Comment (duboisp) [inline site/pages/search/index.hbs]: Replace this: "https://www.canada.ca/en/sr/srb/sra.html" by "[URL of the advance search page]"

File: site/pages/search/results-en.hbs [hunk 5]
```diff
@@ -200,3 +200,8 @@
...
+	<p><a href="https://www.canada.ca/en/sr/srb/sra.html">Advanced search</a></p>
```
Comment (duboisp) [inline site/pages/search/results-en.hbs]: Replace this: "https://www.canada.ca/en/sr/srb/sra.html" by "#URL-of-the-advance-search-page"

File: site/pages/search/results-fr.hbs [hunk 6]
```diff
@@ -86,3 +86,8 @@
 		</ul>
 	</div>
 </section>
+
+{{!-- Advanced search link --}}
+<div>
+	<p><a href="https://www.canada.ca/fr/sr/srb/sra.html">Recherche avancée</a></p>
```
Comment (duboisp) [inline site/pages/search/results-fr.hbs]: Remove the empty div and replace this: "https://www.canada.ca/fr/sr/srb/sra.html" by "#URL-of-the-advance-search-page"

**Comment by coprisanu** [comment]
I would still have a suggestion, to move advanced link centrally.

**Comment by namjohn920** [comment]
> I would still have a suggestion, to move advanced link centrally.

Hi Christian, change was made accordingly.

**Comment by duboisp** [comment]
@Borges-Porteous @coprisanu 

I would like to propose a link text that is more phrasing style instead of be simple word. Something like:
* Perform an advanced search
* Effectuer une recherche avancée

And I am suggesting to make the text link a little bit smaller than normal text by using the "small" element.

What do you think? I think it will make sense where it is located and it will kind of reproduce how Github do their "ProTip!" at the bottom of every of their pages.

**Comment by delisma** [comment]
> @Borges-Porteous @coprisanu
> 
> I would like to propose a link text that is more phrasing style instead of be simple word. Something like:
> 
> * Perform an advanced search
> * Effectuer une recherche avancée
> 
> And I am suggesting to make the text link a little bit smaller than normal text by using the "small" element.
> 
> What do you think? I think it will make sense where it is located and it will kind of reproduce how Github do their "ProTip!" at the bottom of every of their pages.

The suggestion sounds found for TBS. We could go ahead with the merge after all requested changes are completed.

**Comment by coprisanu** [comment]
> 
> 
> @Borges-Porteous @coprisanu
> 
> I would like to propose a link text that is more phrasing style instead of be simple word. Something like:
> 
>     * Perform an advanced search
> 
>     * Effectuer une recherche avancée
> 
> 
> And I am suggesting to make the text link a little bit smaller than normal text by using the "small" element.
> 
> What do you think? I think it will make sense where it is located and it will kind of reproduce how Github do their "ProTip!" at the bottom of every of their pages.

Sounds like a good idea. we should apply it.

**Comment by namjohn920** [comment]
@coprisanu @duboisp 
Requested changes are applied accordingly.

**Comment by coprisanu** [comment]
> 
> 
> @coprisanu @duboisp
> Requested changes are applied accordingly.

Thank you @namjohn920  for making those change, it is looking better now. I was wondering if it would be possible to decrease the space between the pagination bar and the link to advanced search.

**Comment by namjohn920** [comment]
> > @coprisanu @duboisp
> > Requested changes are applied accordingly.
> 
> Thank you @namjohn920 for making those changes, it is looking better now. I was wondering if it would be possible to decrease the space between the pagination bar and the link to advanced search.

@coprisanu @duboisp 
.mrgn-bttm-0 add on pagination. The distance between pagination and advanced search link looks good now.

**Comment by duboisp** [comment]
@namjohn920 please revert the change as it was when @delisma approved this request. Thank you.

@coprisanu 
The spacing between the horizontal line and the pagination and the spacing between the pagination and the advance search link is the same. The advance search link are not part of the pagination component in that template.

**Comment by namjohn920** [comment]
@duboisp 
Changes for margin discarded.

**Comment by duboisp** [comment]
@namjohn920 can you rebase and update your PR? The build should pass now

**Pull request merged**